frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilderModal/MagicFormBuilderModalContent/GifLoadingScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, ModalBody } from '@chakra-ui/react'
+
+import { LottieAnimation } from '~templates/LottieAnimation'
+
+import updatedLoadingAnimation from './assets/updatedLoadingAnimation.json'
+
+export const MagicFormBuilderGifLoadingScreen = (): JSX.Element => {
+  return (
+    <>
+      <ModalBody>
+        <Box display="flex" justifyContent="center">
+          <LottieAnimation
+            animationData={updatedLoadingAnimation}
+            width="47.4rem"
+            height="22rem"
+          ></LottieAnimation>
```

Comment:
nit: this can be `<LottieAnimation animationData= .... />` (i.e. we can use the self-closing syntax here)